### PR TITLE
Updated README to match new transferred repository

### PR DESCRIPTION
**Pull request by BorntraegerMarc**
This change is [Reviewable](https://reviewable.io/reviews/platosha/angular-polymer/125)

### CODE REVIEW

**Comment by BorntraegerMarc** [comment]
Before merge I would like to get input from @platosha 

Also: Should I open a issue to transfer all the resources and documentation from vaadin? Or do you have another plan on how it should be handled?

**Comment by BorntraegerMarc** [comment]
Here is the stuff I saw that needs to be transferred:

1. https://www.npmjs.com/package/@vaadin/angular2-polymer
2. https://vaadin.com/docs/-/part/elements/angular2-polymer/overview.html
3. http://demo.vaadin.com/expense-manager-ng/ -> Not sure about this one. Do we want to create a new demo? I could share some source code that I am personally using...

File: README.md [hunk 1]
```diff
@@ -48,36 +38,32 @@ class AppComponent {
 
 See the overview for a [quick start](https://vaadin.com/docs/-/part/elements/angular2-polymer/overview.html).
 
-See the [tutorial](https://vaadin.com/docs/-/part/elements/angular2-polymer/tutorial-index.html) for complete instructions on how to use `angular2-polymer` and how to build a working application with Angular 2 data binding and routes.
+See the [tutorial](https://vaadin.com/docs/-/part/elements/angular2-polymer/tutorial-index.html) for complete instructions on how to use `angular-polymer` and how to build a working application with Angular data binding and routes.
 
 If you are using [Angular CLI](https://github.com/angular/angular-cli) in your project, see the specific [document](https://vaadin.com/docs/-/part/elements/angular2-polymer/ng-cli-webpack.html) for projects created with the `ng` command line utility.
 
 ## Demo app
```
Comment (platosha) [inline README.md]: This section should be removed, as the demo app is also outdated and could be unpublished soon.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,13 @@
...
-# Angular2-Polymer
-
-`angular2-polymer` is a directive factory that aims at bridging the gaps between using [Polymer](https://www.polymer-project.org) based Web Components in [Angular 2](https://angular.io/) applications.
+`angular-polymer` is a directive factory that aims at bridging the gaps between using [Polymer](https://www.polymer-project.org) based Web Components in [Angular](https://angular.io/) applications.
 
 ```typescript
 import { NgModule, CUSTOM_ELEMENTS_SCHEMA, Component } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-import { PolymerElement } from '@vaadin/angular2-polymer';
+import { PolymerElement } from '@platosha/angular-polymer';
```
Comment (platosha) [inline README.md]: This is too soon. The old package is still available, while the package with this name was not published on NPM still.

Btw, I don’t think `@platosha` should be included in the npm package name.

File: README.md [hunk 3]
```diff
@@ -48,36 +38,32 @@ class AppComponent {
 
 See the overview for a [quick start](https://vaadin.com/docs/-/part/elements/angular2-polymer/overview.html).
```
Comment (platosha) [inline README.md]: Docs on vaadin.com are outdated and could be unpublished soon. Let’s change the documentation links to point to this repo instead.

File: README.md [hunk 4]
```diff
@@ -48,36 +38,32 @@ class AppComponent {
...
 
-See the [tutorial](https://vaadin.com/docs/-/part/elements/angular2-polymer/tutorial-index.html) for complete instructions on how to use `angular2-polymer` and how to build a working application with Angular 2 data binding and routes.
+See the [tutorial](https://vaadin.com/docs/-/part/elements/angular2-polymer/tutorial-index.html) for complete instructions on how to use `angular-polymer` and how to build a working application with Angular data binding and routes.
```
Comment (platosha) [inline README.md]: Same with the doc link here.

**Review by platosha — CHANGES_REQUESTED** [review]

**Comment by BorntraegerMarc** [comment]
Review status: 0 of 1 files reviewed at latest revision, 4 unresolved discussions.

---

*[README.md, line 10 at r1](https://reviewable.io:443/reviews/platosha/angular-polymer/125#-KcNY5x3PdhJ_468Zqqb-r1-10:-KcNbg51jL_Ath1cVzVD:b-896fix) ([raw file](https://github.com/platosha/angular-polymer/blob/83ceb461aeea50f876c192ead669374d60276b1b/README.md#L10)):*
<details><summary><i>Previously, platosha (Anton Platonov) wrote…</i></summary><blockquote>

This is too soon. The old package is still available, while the package with this name was not published on NPM still.

Btw, I don’t think `@platosha` should be included in the npm package name.
</blockquote></details>

Done.

---

*[README.md, line 39 at r1](https://reviewable.io:443/reviews/platosha/angular-polymer/125#-KcNY5x3PdhJ_468Zqqb-r1-39:-KcNcPMwb5o-v-n2e4dA:b-896fix) ([raw file](https://github.com/platosha/angular-polymer/blob/83ceb461aeea50f876c192ead669374d60276b1b/README.md#L39)):*
<details><summary><i>Previously, platosha (Anton Platonov) wrote…</i></summary><blockquote>

Docs on vaadin.com are outdated and could be unpublished soon. Let’s change the documentation links to point to this repo instead.
</blockquote></details>

Done.

---

*[README.md, line 45 at r1](https://reviewable.io:443/reviews/platosha/angular-polymer/125#-KcNY5x3PdhJ_468Zqqb-r1-45:-KcNdpl6gj6WafFmLCip:bmjfgtm) ([raw file](https://github.com/platosha/angular-polymer/blob/83ceb461aeea50f876c192ead669374d60276b1b/README.md#L45)):*
<details><summary><i>Previously, platosha (Anton Platonov) wrote…</i></summary><blockquote>

This section should be removed, as the demo app is also outdated and could be unpublished soon.
</blockquote></details>

I've added the page https://github.com/platosha/angular-polymer/wiki/Using-Polymer-Elements-in-Angular-CLI-Webpack-Applications
I'm not sure if this is the way to go. Please give me a quick feedback. If yes I can migrate all the docs...

Maybe we should start migrating this documents so that we at least have a base line?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/platosha/angular-polymer/125)*

**Comment by BorntraegerMarc** [comment]
Review status: 0 of 1 files reviewed at latest revision, 4 unresolved discussions.

---

*[README.md, line 41 at r1](https://reviewable.io:443/reviews/platosha/angular-polymer/125#-KcNY5x3PdhJ_468Zqqb-r1-41:-KcNeeSmAf_QTqHZQLQN:b-896fix) ([raw file](https://github.com/platosha/angular-polymer/blob/83ceb461aeea50f876c192ead669374d60276b1b/README.md#L41)):*
<details><summary><i>Previously, platosha (Anton Platonov) wrote…</i></summary><blockquote>

Same with the doc link here.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/platosha/angular-polymer/125)*

**Comment by BorntraegerMarc** [comment]
@platosha Could you give me another review please? :)

**Comment by BorntraegerMarc** [comment]
Review status: 0 of 1 files reviewed at latest revision, 4 unresolved discussions.

---

*[README.md, line 45 at r1](https://reviewable.io:443/reviews/platosha/angular-polymer/125#-KcNY5x3PdhJ_468Zqqb-r1-45:-KcNr2t0UgLjTidk3eNP:bst4v77) ([raw file](https://github.com/platosha/angular-polymer/blob/83ceb461aeea50f876c192ead669374d60276b1b/README.md#L45)):*
<details><summary><i>Previously, BorntraegerMarc (Marc Bornträger) wrote…</i></summary><blockquote>

I've added the page https://github.com/platosha/angular-polymer/wiki/Using-Polymer-Elements-in-Angular-CLI-Webpack-Applications
I'm not sure if this is the way to go. Please give me a quick feedback. If yes I can migrate all the docs...

Maybe we should start migrating this documents so that we at least have a base line?
</blockquote></details>

OK, I've updated the location of the doc. I missed the doc folder in the repo before...

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/platosha/angular-polymer/125)*

**Comment by platosha** [comment]
:lgtm:

I’ll update the links shortly after merging.

> https://www.npmjs.com/package/@vaadin/angular2-polymer

We’d better release under different package name in future. I have registered https://www.npmjs.com/package/angular-polymer already.

> https://vaadin.com/docs/-/part/elements/angular2-polymer/overview.html

This repo contains all the documentation content. We can move it to wiki or update as it is.
 
> http://demo.vaadin.com/expense-manager-ng/

I think it’s better to create a new demo, something more lightweight.

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/platosha/angular-polymer/125#-:-KcNtAASVoU2Q-D15n85:bunhefm)*

**Pull request merged**